docs/user-guide/tuning.rst

```diff
@@ -1,5 +1,5 @@
-Tuning and Scaling
-==================
+Tuning, and Monitoring
```

Comment:
In an intermediate change it got added when this became a list of 3. The comma doesn't belong here.